### PR TITLE
Linter fixes

### LINKED ISSUE #3079: Fix linter warnings for accessibility
When executing the "npx ng lint --fix" command several accessibility related linting warnings are reported. We should fix them. 
If they are fixed the behaviour of the client should change (at least slightly) to be more accessible.

What was changed or what different behaviour is to be expected should be documented here or in the PR which fixes the warnings

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #3079

### CODE REVIEW

File: client/src/app/ui/modules/sorting/modules/sorting-list/components/sorting-list/sorting-list.component.html [hunk 1]
```diff
@@ -9,8 +9,10 @@
             cdkDrag
             class="line margin-bottom-5"
             [ngClass]="isSelectedRow(i) ? 'backgroundColorSelected' : 'backgroundColorLight'"
+            [tabindex]="0"
             (cdkDragStarted)="dragStarted(i)"
             (click)="onItemClick($event, i)"
+            (keyup)="onItemClick($event, i)"
```
Comment (Elblinator) [inline client/src/app/ui/modules/sorting/modules/sorting-list/components/sorting-list/sorting-list.component.html:15]: I assume with this change it should be possible to change the order of the os-sorting-lists with just the keyboard
Please explain with what actions/keys it should be possible to change the order
I tried enter, space, arrow keys and ctrl but no item I focus moved in any direction

Furthermore, I was not able to focus the agenda-items and category-items at all

File: client/src/app/ui/modules/grid/grid/grid.component.html [hunk 2]
```diff
@@ -7,7 +7,7 @@ <h3>{{ title }}</h3>
 
 <div class="os-grid" [ngClass]="{ 'no-space': noSpace }">
     @for (tile of tiles; track tile) {
-        <div tabindex="-1" [ngClass]="tile.classes" (click)="tile.onClick($event)">
+        <div tabindex="0" [ngClass]="tile.classes" (click)="tile.onClick($event)" (keyup.enter)="tile.onClick($event)">
```
Comment (Elblinator) [inline client/src/app/ui/modules/grid/grid/grid.component.html:10]: os-gris:
motion-list -> works
meeting-settings -> now has an additional empty focusable item
projector-detail -> now has an additional empty focusable item

Please remove the additional item which is now focusable 
For the projector detail it is just ugly
For the meting settings this is confusing and will be frustrating if used. The visuals between the item which helps you and the one which does nothing is almost identical

**Review by Elblinator — CHANGES_REQUESTED** [review]

File: client/src/app/ui/modules/sorting/modules/sorting-list/components/sorting-list/sorting-list.component.html (hunk 1 above)
Comment (bastianjoel) [reply client/src/app/ui/modules/sorting/modules/sorting-list/components/sorting-list/sorting-list.component.html:15]: No, this is to select multiple Items.

File: client/.eslintrc.js [hunk 3]
```diff
@@ -102,12 +102,9 @@ module.exports = {
                 '@angular-eslint/template/attributes-order': ['error', {
                     alphabetical: true
                 }],
+                '@angular-eslint/template/prefer-control-flow': ['error'],
                 // Should be switched to error
-                '@angular-eslint/template/click-events-have-key-events': ['warn'],
```
[the remark this answers is not in the document]
Comment (bastianjoel) [reply client/.eslintrc.js:106]: They are part of `plugin:@angular-eslint/template/accessibility`.

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-manage-title/motion-manage-title.component.html [hunk 4]
```diff
@@ -6,13 +6,17 @@
...
                             (click)="gotoChangeRecommendation(titleChangeRecommendation)"
+                            (keyup.enter)="gotoChangeRecommendation(titleChangeRecommendation)"
                         ></span>
                     }
                     <span
                         *osPerms="permission.motionCanManage; and: !titleChangeRecommendation"
                         class="change-title"
+                        tabindex="0"
                         (click)="createTitleChangeRecommendation()"
+                        (keyup.enter)="createTitleChangeRecommendation()"
```
[the remark this answers is not in the document]
Comment (bastianjoel) [reply client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-manage-title/motion-manage-title.component.html:19]: Seems to be not possible the way this is currently implemented.

**Review by Elblinator — CHANGES_REQUESTED** [review]
lists where the edit dialog opens with space:
- theme-list
- participant-list

lists where the edit dialog opens with enter:
- agenda-item-list
- motion-list
Please change every list to the space version 

Additionally (I'll create it's own Issue for the following if that should be done in a different PR):
in these lists please stop the list from scrolling down while pressing space
Repo:
- have a list which exceeds the screen space
- focus on a list element (e.g. in the agenda list the title from the item)
- press space

**Review by Elblinator — APPROVED** [review]

**Pull request merged**